openslides/motions/static/js/motions/site.js

```diff
@@ -1008,6 +1009,19 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
         $scope.delete = function (category) {
             Category.destroy(category.id);
         };
+
+        // number all motions in selected category
+        $scope.numbering = function (category) {
+            $http.patch('/rest/motions/category/' + category.id + '/numbering/', {})
```

Comment:
With this patch request I get a 405
`HTTP PATCH /rest/motions/category/3/numbering/ 405 `
What's the right syntax to start numbering? Please add these syntax to docstring.